### ISSUE TITLE
implementation of #107 breaks rendering

### ISSUE THREAD

**Issue by stleusc** [mapsforge/vtm#128]
The implementation of #107 breaks rendering (at least for me, using demo app).
Before the change lines were rendered correctly, after the change not anymore.

Was that fixed based on a real issue? Or more a preventive measure?

Attached screenshots of before and after (using latest HEAD, minus this change)

Bad
![screenshot_20160813-101627](https://cloud.githubusercontent.com/assets/1132106/17643677/467e95c2-613f-11e6-93a8-b67fc1bc2acf.png)
![screenshot_20160813-101636](https://cloud.githubusercontent.com/assets/1132106/17643678/467fadd6-613f-11e6-9606-897007d63c08.png)
```java
if (line.width < 1.5/* || line.fixed*/) {
```

Good
![screenshot_20160813-101911](https://cloud.githubusercontent.com/assets/1132106/17643685/6e2d61a2-613f-11e6-8036-1499ba91836c.png)
![screenshot_20160813-101918](https://cloud.githubusercontent.com/assets/1132106/17643686/6e305d58-613f-11e6-85c6-9e19be9b60aa.png)
```java
if (lb.scale < 1.5/* || line.fixed*/) {
```

I think #107 should be reverted since it seems to break more than it fixes!

**Comment by devemux86** [comment]
Thanks for the report.
Indeed we have a side effect here.

The change in #107 was because with previous code the Cap mode was never used.

**Comment by devemux86** [comment]
I pushed an update for the Cap handling in `LineBucket`.

**Comment by devemux86** [comment]
Fixed via 0d64dfedd4e5da8a86b6172d137c446414cc05ec.
